x-pack/test/functional/services/ml/data_frame_analytics_results.ts

```diff
@@ -61,6 +67,209 @@ export function MachineLearningDataFrameAnalyticsResultsProvider(
       });
     },
 
+    async assertResultsTablePreviewHistogramChartButtonCheckState(expectedCheckState: boolean) {
+      const actualCheckState =
+        (await testSubjects.getAttribute(
+          'mlExplorationDataGridHistogramButton',
+          'aria-pressed'
+        )) === 'true';
+      expect(actualCheckState).to.eql(
+        expectedCheckState,
+        `Chart histogram button check state should be '${expectedCheckState}' (got '${actualCheckState}')`
+      );
+    },
+
+    async enableResultsTablePreviewHistogramCharts(expectedDefaultButtonState: boolean) {
```

Comment:
I'd suggest to rename the parameter for clarity: this is not the default / initial state of the button but the desired state after this method has finished its work. So maybe just remove the `Default` part of the name?

Reply:
Updated [here](https://github.com/elastic/kibana/pull/125162/commits/7b6a8cdc9a228f5c5e1c2baca9c82ce7df4424c2)